frontend/src/components/Home/ItemCard.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import { Typography, Grid, Button, CardHeader, CardMedia } from '@mui/material';
+
+class ItemCard extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    // is it a joke? an image? a taunt? return the appropriate card
+    _resolveItemType = (type, payload) => {
+        if (type === 'joke') {
+            return (
+                <div>
+                    <Typography sx={{ ...styles.title }} color="textSecondary" gutterBottom>
+                        {payload.leadup}
+                    </Typography>
+                    <Typography sx={{ ...styles.title }} color="textSecondary" gutterBottom>
+                        {payload.punchline}
+                    </Typography>
+                </div>
+            )
+        }
+    }
+
+
+    render() {
+        const { id, type, payload } = this.props;
+
+        return (
+            <Card>
+                <CardHeader
+                    title={type + '#' + id}
+                />
+                <CardContent>
+                   {this._resolveItemType(type, payload)}
```

Comment:
IMO should use a formatter to keep styles consistent. Default vscode formatter is probably good enough for now.

Reply:
yep, longer term has a separate card https://github.com/emseers/eelbot-ui/issues/11

Reply:
should be fixed now